web/src/features/time/TimeHeader.tsx

```diff
@@ -1,32 +1,44 @@
 import useGetState from 'api/getState';
 import Badge from 'components/Badge';
+import { NewFeaturePopover } from 'components/NewFeaturePopover/NewFeaturePopover';
+import { NewFeaturePopoverContent } from 'components/NewFeaturePopover/NewFeaturePopoverContent';
 import { FormattedTime } from 'components/Time';
-import { useAtomValue } from 'jotai';
+import { loadingMapAtom } from 'features/map/mapAtoms';
+import { useAtom, useAtomValue } from 'jotai';
 import { useTranslation } from 'react-i18next';
 import { selectedDatetimeIndexAtom, timeAverageAtom } from 'utils/state/atoms';
 
 export default function TimeHeader() {
   const { t, i18n } = useTranslation();
   const timeAverage = useAtomValue(timeAverageAtom);
   const selectedDatetime = useAtomValue(selectedDatetimeIndexAtom);
-  const { isLoading } = useGetState();
+  const { isLoading: isLoadingData } = useGetState();
+  const [isLoadingMap] = useAtom(loadingMapAtom);
+
+  const allLoaded = !isLoadingMap && !isLoadingData;
 
   return (
     <div className="flex min-h-6 flex-row items-center">
       <h3 className="grow select-none text-left">
         {t(`time-controller.title.${timeAverage}`)}
       </h3>
-      {!isLoading && (
-        <Badge
-          pillText={
-            <FormattedTime
-              datetime={selectedDatetime.datetime}
-              language={i18n.languages[0]}
-              timeAverage={timeAverage}
-            />
-          }
-          type="success"
-        />
+      {allLoaded && (
+        <NewFeaturePopover
+          side="top"
+          content={<NewFeaturePopoverContent />}
+          portal={false}
+        >
```

Comment:
This should not be showing until we actually implement it. Meanwhile, should we just add it in a FF or what do you think? 

Also, do you think we should add an end date for when it should not show up for any users anymore? e.g a month after the release of the new feature?